src/status_im/ui/components/toolbar/view.cljs

```diff
@@ -118,8 +118,9 @@
 
 (defn simple-toolbar
   "A simple toolbar whose content is a single line text"
-  ([] (simple-toolbar nil))
-  ([title] (toolbar nil default-nav-back [content-title title])))
+  ([] (simple-toolbar nil nil))
+  ([title] (simple-toolbar nil title))
+  ([style title] (toolbar style default-nav-back [content-title title])))
```

Comment:
Don't change order of args in multi-arity functions, break expectations and easily leads to bugs.

Reply:
That's a component, if we follow that logic we should rewrite all the components that take an optional style ?

Reply:
Also it's a pretty urgent bug to fix because all the builds from current develop are untestable until this is merged

Reply:
Maybe an assert on the first argument `(or (nil? style) (map? style))` would be enough to prevent bugs ?

Reply:
> rewrite all the components that take an optional style ?

Idiomatic in Clojure to deal with optional stuff is to append positionally, use a map, or `& opts`. I don't think we should generally let ad hoc JS convention seep into Clojure more than necessary.

> Also it's a pretty urgent bug to fix because all the builds from current develop are untestable until this is merged
  
Ok, I didn't realize this. And seems like we use this component a bunch so less likely to break things by this change. Oh well.

Reply:
>  don't think we should generally let ad hoc JS convention
that's rather hiccup thing, no? Ability to skip styles map. And it's quite convenient imo.

Reply:
@rasom This is true. To me it is a bit different because it's always in Hiccup vector forms, so there isn't a lot of room for variability. There is only `[:foo ...` or `[:foo style ...`. At least AFAIR, might be wrong. In this case, and similar cases in this namespace, there are more positional arguments and we do it for some components/fns but not all, etc.